src/Controls/src/Core/Handlers/Items/GroupableItemsViewHandler.Windows.cs

```diff
@@ -18,14 +18,12 @@ public static void MapIsGrouped(GroupableItemsViewHandler<TItemsView> handler, G
 
 		protected override CollectionViewSource CreateCollectionViewSource()
 		{
-			if (ItemsView != null && ItemsView.IsGrouped)
+			var itemsSource = Element.ItemsSource;
+			if (ItemsView is not null && ItemsView.IsGrouped && itemsSource is not null)
```

Comment:
Is this change necessary? What if ItemsSource is set to null?

Reply:
Yes, if ItemsSource is updated to null, it throws a null exception while creating the source using TemplatedItemSourceFactory, so this change is necessary.